doc/source/whatsnew/v1.1.0.rst

```diff
@@ -715,6 +715,43 @@ apply and applymap on ``DataFrame`` evaluates first row/column only once
     df.apply(func, axis=1)
 
 
+.. _whatsnew_110.api_breaking.groupby_results_los_as_index_false:
+
+:meth:`DataFrameGroupby.agg` lost results with ``as_index`` ``False`` when relabeling columns
```

Comment:
can you move this right under the other section that deals with `as_index=False`

Reply:
Done